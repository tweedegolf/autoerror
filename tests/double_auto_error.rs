use autoerror::AutoError;

#[derive(Debug, AutoError)]
enum Error {
    #[auto_error(err=true)]
    #[auto_error()]
    A(std::io::Error),
    B(std::fmt::Error),
}

pub fn main() {
}
