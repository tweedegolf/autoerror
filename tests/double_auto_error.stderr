error: Duplicate occurence of auto_error attribute
 --> $DIR/double_auto_error.rs:6:5
  |
6 |     #[auto_error()]
  |     ^^^^^^^^^^^^^^^
